### PR TITLE
Feature/what is meta data

### PR DESCRIPTION
**Pull request by joshLong145**
## Changes
* Addition of `metadata` field in `what_is` definition for user defined meta data content.
* Addition of `metadata` as a request parameter for `MsgCreateSchema`
* Change to schema cid being a `guid` to `cid`

## API Updates
* Update to `MsgCreateSchema` request model for addition of metadata

## Checklist
* [ ] Unit tests
* [x] Documentation

<sub> Mention [stepsize] in a comment if you'd like to report some technical debt. See examples [here](https://app.stepsize.com/api/demo-pr-redirect).</sub>

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1202528620406598/1202766471605524) by [Unito](https://www.unito.io)

### CODE REVIEW

File: x/schema/keeper/msg_server_schema.go [hunk 1]
```diff
@@ -63,6 +61,7 @@ func (k msgServer) CreateSchema(goCtx context.Context, msg *types.MsgCreateSchem
 		Schema:    &schema,
 		Timestamp: time.Now().Unix(),
 		IsActive:  true,
+		Metadata:  make(map[string]string),
```
Comment (mcjcloud) [inline x/schema/keeper/msg_server_schema.go]: You should populate this with the metadata from the request.

**Review by mcjcloud — CHANGES_REQUESTED** [review]

File: x/schema/keeper/msg_server_schema.go (hunk 1 above)
Comment (joshLong145) [reply x/schema/keeper/msg_server_schema.go]: egg on my face, that would be nice lol

**Review by mcjcloud — APPROVED** [review]
Lgtm

**Comment by ntindle** [comment]
@mcjcloud woukd you mind building the motor changes into this PR too?

**Review by ntindle — APPROVED** [review]

**Pull request merged**